### DISCUSSION TITLE
Game File Transfer Over Local Network

### DISCUSSION THREAD

**Discussion by reynonic** [Foxlider/FASTER#133] [Ideas]
It would be nice to allow the new Game File Transfer Over Local Network option, if available. I don't run my server 24/7, so it would be nice to get the updates over LAN, rather than having to deal with any throttling. Thanks for your consideration. Great program! Thanks for all of the hard work!
![image](https://github.com/Foxlider/FASTER/assets/39746322/137066a5-9294-419b-a542-a8429e996bd3)

**Comment by Foxlider** [comment]
I am really not sure that's something possible... 
I believe it's Steam Client to Steam Client...
